extensions/json.py

```
@@ -0,0 +1,41 @@
+#
+
+# JSON -> MODULES
+import datetime
+import json
+
+# JSON -> CLASS
+class namespace:
+    # CLASS -> VARIABLES
+    data: dict | int | float | str | bool | list | None
+    # CLASS -> INIT
+    async def init(self, request: Request) -> Self: return self
```

Ruff (0.12.2)

`Request` may be undefined, or defined from star imports

(F405)